projects/vdk-plugins/vdk-control-api-auth/src/vdk/plugin/control_api_auth/auth_utils.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+from configparser import ConfigParser
+from configparser import MissingSectionHeaderError
+
+from vdk.plugin.control_api_auth.auth_exception import VDKAuthException
+
+
+log = logging.getLogger(__name__)
+
+
+def read_config_ini_file(
```

Comment:
what's the intended usage? didn't find any usages (except in test utils, where a duplicate's in usage )

Reply:
Oopsie, good catch. This one was supposed to be part of a separate PR. Removed.